react/components/DatePicker/index.js

```diff
@@ -123,6 +159,8 @@ DatePicker.propTypes = {
   excludeDates: PropTypes.arrayOf(PropTypes.instanceOf(Date)),
   /** Times to be excluded  */
   excludeTimes: PropTypes.arrayOf(PropTypes.instanceOf(Date)),
+  /** Whether or not the popper should be flipped vertically */
+  flipped: PropTypes.bool,
```

Comment:
this could be something like, `direction: up|down`, or `verticalAlignment: top|bottom`, or something of the sort

(when there is no "right" value, or there is no clear semantic of "on/off", it's generally better to use an enum rather than a boolean)

Reply:
Also, when I read `flipped` I imagine the **DatePicker** upside down, haha